source/includes/fact-iso-date-objects.rst

```diff
@@ -1,5 +1,31 @@
-The argument can be any valid :ref:`expression
-<aggregation-expressions>` that resolves to a
-:doc:`BSON ISODate object </reference/bson-types>`,
-a :doc:`BSON Timestamp object </reference/bson-types>`,
-or a :doc:`Date object </reference/method/Date/>`.
+The argument must be a valid :ref:`expression
+<aggregation-expressions>` that resolves to one of the following:
+
+- A :ref:`Date <document-bson-type-date>` or
+  :ref:`Timestamp <document-bson-type-timestamp>`.
+
+- A document of the following form:
+
+  .. code-block:: javascript
+
+     { date: <dateExpression>, timezone: <tzExpression> }
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 20 80
+
+     * - Field
+
+       - Description
+
+     * - ``date``
+
+       - The date to which the operator is applied.
+         ``<dateExpression>`` must be a valid :ref:`expression
+         <aggregation-expressions>` that resolves to either a
+         :ref:`Date <document-bson-type-date>` or a
+         :ref:`Timestamp <document-bson-type-timestamp>`.
+
+     * - ``timezone``
+
+       - .. include:: /includes/fact-timezone-description.rst
```

Comment:
could we add a versionadded for the timezone as when 3.6 goes live, this page will show up on
docs.mongodb.com/manual
so, even people with older version may look at this and think the timezone is allowed field.